src/users/enums/user-role.ts

```diff
@@ -0,0 +1,6 @@
+export enum UserRole {
```

Comment:
Should it be used in seed.ts ?

Reply:
As discussed, the `src` directory will not be copied in the Docker image so we can't reference it from the `seed.ts` file. I think we are obliged to use as it is now.